src/AMSlib/AMS.cpp

```diff
@@ -87,7 +87,12 @@ AMSExecutor AMSCreateExecutor(const AMSConfig config)
                                      config.SPath,
                                      config.DBPath,
                                      config.dbType,
+<<<<<<< HEAD:src/AMSlib/AMS.cpp
                                      config.device,
+=======
```

Comment:
@alizalisan over here you need to pass as arguments both config.physics_device and config.ml_device. These will instruct the workflow to schedule each computation accordingly.

Reply:
Fixed.